packages/api-explorer/src/JsonForm/refReplacer.js

```diff
@@ -34,11 +34,11 @@ function isChildADefinition (currentKey, obj) {
 
 export default function replaceRefs(obj)  {
   const searchStr = '$ref'
-
-  if (!obj) {
+  
+  if (obj === undefined) {
     return;
   }
-  if (typeof obj === 'string') {
+  if (['string', 'boolean', 'number'].includes(typeof obj)) {
```

Comment:
This should not be `typeof obj != object`?

Reply:
you're right

Reply:
done